src/Kernel32/storebanned/Kernel32+DISK_GEOMETRY.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="DISK_GEOMETRY"/> nested type.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// Describes the geometry of disk devices and media.
+        /// </summary>
+        /// <seealso href="https://docs.microsoft.com/en-us/windows/win32/api/winioctl/ns-winioctl-disk_geometry"/>
+        public struct DISK_GEOMETRY
+        {
+            /// <summary>
+            /// The number of cylinders.
+            /// </summary>
+            public long Cylinders;
```

Comment:
Oddly enough, this is typed as [LARGE_INTEGER ](https://webcache.googleusercontent.com/search?q=cache:HVWWFBB64_kJ:https://docs.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-large_integer~r1+&cd=1&hl=en&ct=clnk&gl=be) which is signed.